parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -121,7 +119,9 @@ pub mod token_distribution {
     use super::*;
 
     // 10 million KINT distributed over 4 years
-    pub const INITIAL_ALLOCATION: Balance = 10_000_000_000_000_000_000;
+    // KINT has 12 decimal places, same as KSM
+    // See: https://wiki.polkadot.network/docs/learn-DOT#kusama
+    pub const INITIAL_ALLOCATION: Balance = 10_000_000_u128 * UNITS;
```

Comment:
I think this is now wrong, we should calculate from `UNITS` or add the helper as @sander2 suggests.